schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/convert/fromJsonOpenDataSoft.xsl

```diff
@@ -555,16 +565,18 @@
             </gco:CharacterString>
           </cit:linkage>
           <cit:protocol>
-            <gco:CharacterString>WWW:DOWNLOAD-1.0-http--download</gco:CharacterString>
+            <gco:CharacterString>
+              <xsl:value-of select="concat($format-mimetype-mapping/entry[format=lower-case($format)]/protocol,':', $format-mimetype-mapping/entry[format=lower-case($format)]/mimetype)"/>
+            </gco:CharacterString>
           </cit:protocol>
           <cit:name>
             <gco:CharacterString>
-              <xsl:value-of select="concat('Download as ', upper-case($format))"/>
+              <xsl:value-of select="$format"/>
```

Comment:
Why not write something a bit more explicit? I agree that it's not multilingual though

Reply:
I didn't want to polluate the metadata itself and thought we can do something in the UI if needed.
Otherwise yes, we would need to add a translation and use the metadata language

Reply:
Ok, makes sense I guess.